composer.json

```diff
@@ -11,7 +11,7 @@
   ],
   "require": {
     "php": "^7.0",
-    "laravel/framework": "~5.2.0|~5.3.0|~5.4.0|~5.5.0|~5.6.0|~5.7.0|~5.8.0"
+    "laravel/framework": "~5.2.0|~5.3.0|~5.4.0|~5.5.0|~5.6.0|~5.7.0|~5.8.0|^6.0|^7.0"
```

Comment:
@jakemercurio I think this would be compatible with 8 as well, mind adding it also?

Reply:
Added it in a new PR!